### DISCUSSION TITLE
几分钟就会掉线 一直在重连

### DISCUSSION THREAD

**Discussion by howiezzzz** [natfrp/wiki#424] [使用问题]
![IMG_7711](https://github.com/natfrp/wiki/assets/75358376/d832b910-ad47-4d7c-a4ba-8fe88e673a07)
求助 这大概会是什么问题？

**Comment by fakeboboliu** [comment]
我们对此问题也比较好奇，目前我个人的猜测是：您的运营商可能在尝试关闭我们的 bUDPv2 传输线程（即这个版本更新的高效 UDP 传输）。

您可以尝试覆盖安装 [3.0.2.0](https://nya.globalslb.net/natfrp/client/launcher-windows/3.0.2.0/SakuraLauncher.exe) 版本，这个版本使用效率更低的方案。
